CMakeLists.txt

```diff
@@ -18,7 +18,16 @@ include(Sanitizers)
 include(RealmCore)
 set(REALM_CORE_VERSION "1.5.0" CACHE STRING "")
 use_realm_core(${REALM_CORE_VERSION})
-build_realm_sync(${REALM_SYNC_PREFIX})
+
+option(REALM_ENABLE_SYNC "Build realm object store with sync extensions" OFF)
```

Comment:
Since we're now setting `REALM_ENABLE_SYNC` based on whether or not `REALM_SYNC_PREFIX` was set, I don't think there's any reason for `REALM_ENABLE_SYNC` to be independently configurable. You can likely just remove this line.